src/components/InlineWrapper/InlineWrapper.docs.mdx

```diff
@@ -0,0 +1,23 @@
+import { Story, Preview } from '@storybook/addon-docs/blocks';
+import InlineWrapper from './InlineWrapper';
+
+# InlineWrapper
+
+A stylized box that brings attention to a piece of text.
+
+## Usage
+
+```js
+import { InlineWrapper } from '@quartz/interface/src/components';
+```
+
+## Props
+
+This component accepts children as props and wraps them with a stylized container.
```

Comment:
```suggestion
<Props of={InlineWrapper} />
```
Use the Storybook Props component to generate a table of props (even thought there is only one in this case)

Reply:
You can add a description of the prop in the component's propTypes, see the Button component for an example